src/tools/fuzzing/fuzzing.cpp

```diff
@@ -3287,6 +3287,30 @@ Expression* TranslateToFuzzReader::makeStructSet(Type type) {
   return builder.makeStructSet(fieldIndex, ref, value);
 }
 
+static auto makeArrayBoundsCheck(Expression* ref,
```

Comment:
TIL you can use auto like this and even return and use unnameable types. Neat!

Reply:
Even nicer would be to return an anonymous struct like some languages have... but yes, this is not bad...